site/content/docs/main/resource-filtering.md

```diff
@@ -88,6 +88,12 @@ Wildcard takes precedence when both a wildcard and specific resource are include
   ```bash
   velero backup create <backup-name> --selector <key>=<value>
   ```
+* Include resources that are not matching the selector
+  ```bash
+  velero backup create <backup-name> --selector <key>!=<value>
+  ```
+
+For more information read the [Kuberentes label selector documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors)
```

Comment:
Can you please fix the spelling of `Kubernetes`? Thanks!

Reply:
Updated